src/handlers/package_handler.js

```diff
@@ -60,31 +60,32 @@
   // The endpoint using this function needs an array.
   const packArray = Array.isArray(packObjShort) ? packObjShort : [packObjShort];
 
-  const totalPages = await database.getTotalPackageEstimate();
+  const pagination = await database.getTotalPackageEstimate();
 
-  if (!totalPages.ok) {
+  if (!pagination.ok) {
     logger.generic(
       3,
-      `getPackages-getTotalPackageEstimate Not OK: ${totalPages.content}`
+      `getPackages-getTotalPackageEstimate Not OK: ${JSON.stringify(pagination.content)}`
     );
-    await common.handleError(req, res, totalPages, 1002);
-    return;
-  }
-
-  res.append(
-    "Link",
-    `<${server_url}/api/packages?page=${params.page}&sort=${
-      params.sort
-    }&order=${
-      params.direction
-    }>; rel="self", <${server_url}/api/packages?page=${
-      totalPages.content
-    }&sort=${params.sort}&order=${
-      params.direction
-    }>; rel="last", <${server_url}/api/packages?page=${params.page + 1}&sort=${
-      params.sort
-    }&order=${params.direction}>; rel="next"`
-  );
+    await common.handleError(req, res, pagination, 1002);
+    return;
+  }
+
+  let link = `<${server_url}/api/packages?page=${params.page}&sort=${params.sort}&order=${
+    params.direction
+  }>; rel="self", <${server_url}/api/packages?page=${pagination.content.pages}&sort=${
+    params.sort
+  }&order=${params.direction}>; rel="last"`;
+
+  if (params.page !== pagination.content.pages) {
+    link += `, <${server_url}/api/packages?page=${
+      params.page + 1
+    }&sort=${params.sort}&order=${params.direction}>; rel="next"`
```

Comment:
@confused-Techie I can change it to an `Array.join(', ')` if you'd prefer, but I don't particularly agree why the code is bad.

Reply:
wut? Mr. CodeQL, where do you see an automated semicolon?

Reply:
> wut? Mr. CodeQL, where do you see an automated semicolon?

I think it means the JavaScript compiler will have to insert a semicolon automatically, since there's not one after the template literal

Reply:
Mmmm, yes. Good catch, I'll add those in. Thanks for translating 😅 looks like I need to pull upstream changes too